joomla/joomla-cms
# [4.0] Normalize JS in All Categories views

Fixes https://github.com/joomla/joomla-cms/issues/26695.

### Summary of Changes

Normalizes `categories-default.js` in `com_content`, com_contact` and `com_newsfeeds`, fixing several issues in All Categories views.

### Testing Instructions

Create categories with child categories for mentioned components.
Go to `List All Categories` views in frontend.
Click the `+` button to expand categories.
Check that plus icon changes to minus and that `aria-label` attribute of the button updates.

### Documentation Changes Required

No.
